bezpy/mt/site.py

```diff
@@ -243,8 +243,19 @@ def download_waveforms(self):
 
         # Magnetic Field
         self.waveforms[["FE", "FN", "FZ"]] *= 0.01  # nT
-        # Electric Field
-        self.waveforms[["QN", "QE"]] *= 2.44141221047903e-05  # mV/km
+        # Electric voltage
+        self.waveforms[["QN", "QE"]] *= 2.44141221047903e-06  # mV
```

Comment:
Do you know where this value comes from? If you have a reference that would be helpful to add in as a comment here. Maybe this is even NIMS specific?

Reply:
2.44141221047903e-06 is the NIMS specific value converting from digitized value to the physical value. The unit becomes to mV. 2.44141221047903e-05 is used when the length of the electric dipole is 100m as I understand.